examples/notebooks/Makefile

```diff
@@ -26,3 +26,31 @@ quarto: $(QUARTO_MD_OUTPUTS)
 
 clean:
 	rm -rf $(QUARTO_MD_OUTPUTS) $(QUARTO_FILES_OUTPUTS)
+
+# A straightforward target to enable `make docker TARGET=bash`
+bash:
+	@bash
+
+define VAST_COMPOSE_OVERLAYS
+-f ../../docker/quarto/docker-compose.yaml
+-f ../../docker/quarto/docker-compose.bind.yaml
+-f ../../docker/quarto/docker-compose.vast.yaml
+-f ../../docker/vast/docker-compose.yaml
+endef
+
+export VAST_COMPOSE_OVERLAYS
+# Run this makefile from within the Quarto Docker image, with a VAST server
+# running in another service.
+# You can specify another target using the TARGET variable:
+# make docker TARGET=clean
+docker: 
+	docker compose \
+		$$VAST_COMPOSE_OVERLAYS \
+		up -d vast && \
+	HOST_UID=$$(id -u) HOST_GID=$$(id -g) docker compose \
+		$$VAST_COMPOSE_OVERLAYS \
+		run quarto \
+		make -C examples/notebooks $(TARGET) ; \
```

Comment:
This is where you run `quarter render`, and it's also where you can say `--to markdown` or so.

Reply:
very strangely, when I add `--to markdown` in the makefile:
```Makefile
%.md: %.qmd
ifndef QUARTO
	$(error "quarto not in $$PATH; please install system-wide")
else
	poetry run quarto render --to markdown $<
endif
```

No output is generated, but no error message is generated either 😓. I'll live it like this because this is more a quarto related issue than a Dockerized quarto thing, but still weird.

Reply:
I agree, it's not worth investigating deeper since it's just the example.